packages/lesswrong/components/form-components/UsersListEditor.jsx

```diff
@@ -109,23 +106,21 @@ class UsersListEditor extends Component {
     const { classes, label, currentUser } = this.props
 
     return (
-      <div className={classes.root}>
-        <div className={classes.search}>
-          <Components.ErrorBoundary>
-            <Components.UsersSearchAutoComplete
-              clickAction={this.addUserId}
-              label={label}
-            />
-          </Components.ErrorBoundary>
-          <SortableList
-            axis="xy"
-            items={this.state.userIds}
-            onSortEnd={this.onSortEnd}
-            currentUser={currentUser}
-            removeItem={this.removeUserId}
-            shouldCancelStart={this.shouldCancelStart}
+      <div className={classes.search}>
```

Comment:
General rule for Material-UI is that the root level element has `classes.root` applied to it. So I think we should stick to that.